src/migrations/scripts/1597921613888-initial.ts

```diff
@@ -10,7 +10,6 @@ import { Sequelize as SequelizeTs } from 'sequelize-typescript'
  * createTable "rates", deps: []
  * createTable "rns_transfer", deps: []
  * createTable "rns_domain", deps: []
- * createTable "rns_domain_expiration", deps: [rns_domain]
```

Comment:
I think you would need to create a new migration script for the changes as these changes. @nduchak is this correct?

Reply:
done